### PR TITLE
feature: FormatCredit

### PR DESCRIPTION
**Pull request by AshKyd**
Light:

image
image

---
Night:

image
image

---
Custom:

image
image

---
Legacy format still intact:

image

### CODE REVIEW

File: src/app/components/legacy/FormatCredit/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const LINK_URL = '/news/interactives/';
+const LINK_TRACKER = () => track('format-credit-link', '*');
+
+const FormatCredit = () => {
+  styles.use();
+
+  return html`
+    <div class="FormatCredit--legacy">
+      <p>
+        <span class="FormatCredit__wrapper">
```
Comment (AshKyd) [inline src/app/components/legacy/FormatCredit/index.js:14]: Needed to add a wrapper for the same reason, to revert to original styles without specificity issues.

**Comment by AshKyd** [comment]
Wait, the text colour is wrong. I think it needs to be `var(--od-colour-theme-text-accent)`

Edit: Wait, it is! But it's the wrong colour. Will look into it.

File: src/app/components/FormatCredit/index.lazy.scss [hunk 2]
```diff
@@ -1,23 +1,52 @@
 @import '../../constants';
 
 .FormatCredit {
-  color: $color-black-transparent-75;
-  font-family: var(--od-font-stack-sans);
+  color: $color-black;
```
Comment (jtfell) [inline src/app/components/FormatCredit/index.lazy.scss]: I'd suggest using the `--od` var for this, then you shouldn't need the `invert` override below

Comment (AshKyd) [reply src/app/components/FormatCredit/index.lazy.scss]: Thanks, fixed.

File: src/app/components/FormatCredit/index.lazy.scss [hunk 3]
```diff
@@ -1,23 +1,48 @@
...
-  }
-
-  .FormatCredit > * {
+  .FormatCredit>* {
     color: inherit;
   }
 
-  & a {
-    white-space: nowrap;
+  // pill/tag w specificity to fight Odyssey richtext links
```
Comment (AshKyd) [inline src/app/components/FormatCredit/index.lazy.scss:15]: Fixed the issue with the wrong link colour inherited from Odyssey links.

This issue comes from here: https://github.com/abcnews/odyssey/blob/68b1a7310f1897b5c45135e9502dd67c4fcf7092/src/app/components/utilities/richtext.scss#L402

I'm wondering if we should have a `:not(.custom)` in there so we can style our own UI elements without fighting specificity

Comment (AshKyd) [reply src/app/components/FormatCredit/index.lazy.scss:15]: Ehhh, it can be a later problem. We don't need to solve that now, it's working.

Comment (drzax) [reply src/app/components/FormatCredit/index.lazy.scss:15]: I'm not exactly sure I understand the problem, but I'm happy to put it in the later basket. A convention/system for flagging custom overrides so Odyssey has more options for responding would be handy in future. Needs some thought though.

**Review by jtfell — APPROVED** [review]

**Review by drzax — APPROVED** [review]

**Pull request merged**